guides-and-standards/standards--source-control.md

```diff
@@ -0,0 +1,153 @@
+
+## Core standards
+
+- [Arcadia Science](https://www.arcadiascience.com/) uses [Git](https://git-scm.com/) and [GitHub](https://github.com/) for version control.
+- Repositories on GitHub should be private by default and created under the `Arcadia-Science` organization.
+- Code and documentation should be added to repositories throughout a project, not at the time of publication.
+- Work is done in branches and integrated into the main branch through pull requests.
+- All Pull Requests should be reviewed by at least one other Arcadian and merged by the author of the pull request upon approval.
+- Code associated with a pub is tagged in a release and archived on [Zenodo](https://zenodo.org/).
+
+## Repositories
+
+At Arcadia Science, all GitHub repositories should follow basic guidelines.
+
+- **Repository location**: Repositories for code written at Arcadia should be under the [Arcadia-Science GitHub organization](https://github.com/Arcadia-Science) and not under personal accounts.
+- **Private by default**: Repositories should be private by default. If you need to make a repository public, please consult with the software team.
+- **Repository names:** Repository names are lower case and hyphenated. (e.g. `arcadia-software-handbook`). The names of analysis repositories should be prefixed with the year of the project (e.g. `2024-worm-tracking`).
+- **README**: Repositories should contain a descriptive README document.
+- **License**: Each repository should have a license file. The default license for Arcadia Science repositories is [the MIT license](https://github.com/Arcadia-Science/python-analysis-template/blob/main/LICENSE).
+- **Archiving repositories:** If a repository is no longer useful, make it read-only by archiving it. Follow the instructions [here](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories).
+- **No deleted repositories:** Never delete repositories. This ensures that other members (past and future) of Arcadia can see what worked and what hasn’t.
+
+### Public repositories
+
+Additional standards apply to public repositories to ensure that the code we release is reusable by others and that the computational artifacts we release are reproducible by others. These standards are described in a separate document [here](./standards--public-repos.md).
+
+### Repository templates
+
+We have Arcadia-specific templates for [Python](https://github.com/Arcadia-Science/python-analysis-template), [R](https://github.com/Arcadia-Science/r-analysis-template), [Snakemake](https://github.com/Arcadia-Science/snakemake-template), and [Nextflow](https://github.com/Arcadia-Science/nextflow-template-repository) repositories. It is strongly recommended to start new projects using these templates, as this will mean that your repository will automatically satisfy most of the requirements listed above.
```

Comment:
do we have a python package one now too?

Reply:
yes! added.